exonum/src/sandbox/consensus/block_request.rs

```diff
@@ -218,6 +223,108 @@ fn handle_block_response_with_unknown_tx() {
     ));
 }
 
+/// - should **NOT** process block if tx is incorrect
+/// idea of test is:
+/// - getting Status from other node with later height, send BlockRequest to this node
+/// - receive BlockResponse with unknown tx A
```

Comment:
Nit / future work: Could make sense to add a case when the transaction is known to be incorrect to the node.